ui/tutorial-renderer/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+
+android {
+    defaultConfig {
+        vectorDrawables.useSupportLibrary = true
+
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments = [eventBusIndex: 'org.cru.godtools.tract.TractEventBusIndex']
```

Comment:
you aren't using eventbus, so this isn't necessary. plus it would cause a conflict with the tract renderer if you were using eventbus